lib/server_configfile.ml

```diff
@@ -97,6 +97,18 @@ let check_is_docker_compatible name =
   if not (String.for_all Oca_lib.char_is_docker_compatible name) then
     failwith "name field has to contain only alphanumerical characters and '.'"
 
+let ensure_no_duplicate_switches switches =
```

Comment:
```suggestion
let check_no_duplicate_switches_exists switches =
```
`check` seems more common as a name for a `a -> unit` function that triggers an exception (it's nitpicky)

Reply:
`check` sounds more like `'a -> bool` to me, whereas `ensure` sounds like it'll explode on assertion failures. But ok, how about `assert`? That is fairly well-established to raise on failures.

Reply:
I like assert :+1: Thanks!